From f41adcd85164ef8a0d0918f0508f29dcf3c7467d Mon Sep 17 00:00:00 2001
From: Robert Marshall <rmarshall@redhat.com>
Date: Tue, 26 Jan 2016 10:28:35 -0500
Subject: [PATCH 81/90] Clean up grub-setpassword documentation (#1290799)

The output for --help had some errors. Corrected those and polished the
text to be a little easier to follow. Carried verbage over to man page
to maintain internal consistency.

Resolves: rhbz#1290799
---
 util/grub-setpassword.8  |  2 +-
 util/grub-setpassword.in | 15 +++++++--------
 2 files changed, 8 insertions(+), 9 deletions(-)

diff --git a/util/grub-setpassword.8 b/util/grub-setpassword.8
index 49200a8..dc91dd6 100644
--- a/util/grub-setpassword.8
+++ b/util/grub-setpassword.8
@@ -19,7 +19,7 @@ Display program usage and exit.
 -v, --version
 Display the current version.
 .TP
--o, --output[=\fIDIRECTORY PATH\fR]
+-o, --output=<\fIDIRECTORY\fR>
 Choose the file path to which user.cfg will be written.
 
 .SH SEE ALSO
diff --git a/util/grub-setpassword.in b/util/grub-setpassword.in
index dd7b793..d7924af 100644
--- a/util/grub-setpassword.in
+++ b/util/grub-setpassword.in
@@ -16,15 +16,14 @@ grub_mkpasswd="${bindir}/@grub_mkpasswd_pbkdf2@"
 # Print the usage.
 usage () {
     cat <<EOF
-Usage: $0 [OPTION] [SOURCE]
-Run GRUB script in a Qemu instance.
-
-  -h, --help              print this message and exit
-  -v, --version           print the version information and exit
-  -o, --output_path       choose a custom output path for user.cfg
-
+Usage: $0 [OPTION]
 $0 prompts the user to set a password on the grub bootloader. The password
-is written to a file named user.cfg.
+is written to a file named user.cfg which lives in the GRUB directory
+located by default at ${grubdir}.
+
+  -h, --help                     print this message and exit
+  -v, --version                  print the version information and exit
+  -o, --output_path <DIRECTORY>  put user.cfg in a user-selected directory
 
 Report bugs at https://bugzilla.redhat.com.
 EOF
-- 
2.9.3

